src/Essentials/samples/Samples/View/AccelerometerPage.xaml

```diff
@@ -8,43 +8,29 @@
         <viewmodels:AccelerometerViewModel />
     </views:BasePage.BindingContext>
 
-    <StackLayout>
+    <Grid RowDefinitions="Auto,*">
```

Comment:
In MAUI, a root stack layout will expand beyond the screen bounds, so we need to use a grid.